subprojects/internal-performance-testing/src/test/groovy/org/gradle/performance/results/DefaultPerformanceFlakinessAnalyzerTest.groovy

```diff
@@ -64,6 +64,7 @@ class DefaultPerformanceFlakinessAnalyzerTest extends Specification {
 FROM-BOT
 
 Coordinator url: https://builds.gradle.org/viewLog.html?buildId=${System.getenv("BUILD_ID")}
+Branch: ${System.getProperty('org.gradle.performance.execution.branch')}
```

Comment:
Can we use a system property rule in this test, so we actually set system property to something we can assert?